abel/benchmark.py

```diff
@@ -196,15 +202,13 @@ def absolute_ratio_benchmark(analytical, recon, kind='inverse'):
 
     err = func[mask]/recon[mask]
     return err
-    
+
 
 def main():
     # run some benchmarks!!
-    time = AbelTiming(n=[501]) 
+    time = AbelTiming(n=[301, 501])
```

Comment:
Now that I think about it, maybe make these the defaults for the `AbelTiming.__init__()` function and then just have main run it with the default values.